### PR TITLE
chore: add quality assurance questions

### PR DESCRIPTION
**Pull request by RitaDee**
## Summary of changes
Add quality assurance questions

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/quality-assurance-quiz.ts [hunk 1]
```diff
@@ -182,6 +182,27 @@ const qualityAssuranceQuiz = [
...
+  },
+  {
+    Question:
+      "What is the primary focus of quality assurance (QA) in the Software Development process, as compared to quality control (QC)?",
+    Answer: "QA focuses on proactive processes and adherence to quality standards, while QC is primarily concerned with reactive corrections",
+    Distractor1: "QA focuses on inspection and testing, while QC focuses on documentation, and audits.",
+    Distractor2: "QA and QC implies the same thing and can be used interchangeably.",
+    Distractor3: "QC is perfomed before QA.",
+    Explanation:
+      "QA focuses more on proactive processes while QC focuses more on reactive corrections. QA is concerned with processes and aadherence, while QC is centered on inspections, testing and making corrections.",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:194]:
```suggestion
      "QA focuses more on proactive processes while QC focuses more on reactive corrections. QA is concerned with processes and adherence, while QC is centered on inspections, testing and making corrections.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/quality-assurance-quiz.ts (hunk 1 above)
Comment (RitaDee) [reply src/data/quality-assurance-quiz.ts]: Thank you for the review. I have fixed the typo.

File: src/data/quality-assurance-quiz.ts [hunk 2]
```diff
@@ -182,6 +182,27 @@ const qualityAssuranceQuiz = [
...
+    Answer: "QA focuses on proactive processes and adherence to quality standards, while QC is primarily concerned with reactive corrections",
+    Distractor1: "QA focuses on inspection and testing, while QC focuses on documentation, and audits.",
+    Distractor2: "QA and QC implies the same thing and can be used interchangeably.",
+    Distractor3: "QC is perfomed before QA.",
+    Explanation:
+      "QA focuses more on proactive processes while QC focuses more on reactive corrections. QA is concerned with processes and adherence, while QC is centered on inspections, testing and making corrections.",
+    Link: "https://www.freecodecamp.org/news/how-to-become-a-quality-assurance-engineer-qa-engineer-career-guide/"
+  },
+  {
+    Question: "Which of the following skills is NOT typically required for a QA engineer",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:198]: just spotted this missing punctuation

```suggestion
    Question: "Which of the following skills is NOT typically required for a QA engineer?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Just spotted one last error.
Otherwise it is good to go

File: src/data/quality-assurance-quiz.ts (hunk 2 above)
Comment (RitaDee) [reply src/data/quality-assurance-quiz.ts]: Done

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**